chart/portworx/templates/storage-cluster.yaml

```diff
@@ -191,11 +191,13 @@ spec:
         path: /lib/modules
         type: Directory
   {{- end }}
-  {{- if eq $telemetry true}}
   monitoring:
     telemetry:
+      {{- if eq $telemetry true}}
```

Comment:
Check if telemetry is nil or none here. And in the enabled section use the value of it's not nil.

Reply:
@sharma-tapas Since we expose helm options in catalog UI, if user provides a garbage value (apart from true/false), the same will get used in enabled section. With current approach, only for `telemetry: true` value, telemetry will get enabled and for all rest of the values, (nil, none, garbage), telemetry will be disabled.

Have tried the appraoch here similar to : https://github.com/portworx/ibm-helm/blob/master/chart/portworx/templates/storage-cluster.yaml#L73

Let me know your thoughts on this.